plugins/scripts/inject-config.js

```diff
@@ -1,200 +1,255 @@
 
 const fs = require('fs');
 const path = require('path');
+const elementtree = require('elementtree');
```

Comment:
Is this dep available by default in Cordova installs or should we have a dependency on it?

Reply:
i think that's what Cordova uses to parse plugin.xml and config.xml